10_files_lab/fileslab.txt

```diff
@@ -8,30 +8,49 @@ Online: https://www.tocode.co.il/bundles/unix1/lessons/fileslab
   - .highscore
 
 Write here the commands you used:
+	sudo mkdir lab
+	sudo touch a.out hero.c enemy.c game.c main.c human.h monster.h .highscore
+	
 
 2. "cd" into lab folder and inside create the following subdirectories:
   - Music
   - Misc
   - Drivers
 
 Write here the commands you used:
+	cd lab
+	sudo mkdir Music Misc Drivers
 
 3. Copy the file "/etc/passwd" to each subdirectory created in (2).
 
 Write here the commands you used:
 
+	sudo cp /etc/passwd /home/lab/Music 
```

Comment:
that's correct (except of the sudo which I mentioned above :P), but you can also use the shorthand for your home dir "~"

``` cp /etc/passwd ~/Music ```

(whatever you prefer, it's a matter of style)